app/src/main/res/layout/cuisine_card.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <com.google.android.material.card.MaterialCardView
+        android:layout_width="125dp"
+        android:layout_height="150dp"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:cardCornerRadius="12dp"
+        android:layout_margin="16dp">
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:background="@drawable/img"
+            android:layout_marginStart="8dp">
+
+                <androidx.appcompat.widget.AppCompatTextView
+                    android:id="@+id/cuisine_name"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:text="@string/indian"
+                    android:textSize="14sp"
+                    android:textStyle="bold"
+                    app:layout_constraintBottom_toTopOf="@id/card_text"
+                    app:layout_constraintStart_toStartOf="parent"/>
+                <androidx.appcompat.widget.LinearLayoutCompat
+                    android:id="@+id/card_text"
+                    android:layout_width="match_parent"
+                    android:layout_height="28dp"
+                    android:orientation="horizontal"
+                    app:layout_constraintBottom_toBottomOf="parent"
+                    app:layout_constraintStart_toStartOf="parent">
+                    <androidx.appcompat.widget.AppCompatTextView
+                        android:id="@+id/item_number"
+                        android:layout_width="wrap_content"
+                        android:layout_height="match_parent"
+                        android:paddingEnd="8dp"
+                        android:text="@string/_15"
+                        android:textSize="12sp" />
+                    <androidx.appcompat.widget.AppCompatTextView
+                        android:id="@+id/text_item"
+                        android:layout_width="wrap_content"
+                        android:layout_height="28dp"
+                        android:text="@string/item"
```

Comment:
You should use tools instead of text because it will be used for recyclerview not a static data do it for every text and also the image should be tools